src/OpenURLServer.js

```diff
@@ -41,6 +38,24 @@ class OpenURLServer {
       });
     }
 
+    const svcCfg = cfg.getValues()?.services?.[symbol] ?? cfg.getValues();
+    if (svcCfg?.reqIdHeader) {
+      let fromHeader = ctx.req.headers?.[svcCfg?.reqIdHeader];
+      if (typeof fromHeader === 'string') {
```

Comment:
What else could the header value be, but string?

If it's not  string then nothing at all happens. Is that deliberate? If so, best add a comment to that effect.

Reply:
It's probably a redundant check--I doubt it can return the header as anything but a string so it would be very surprising to see any other value there. I made it explicit because I use a method from the String prototype.